ui/index.html

```diff
@@ -1,39 +1,61 @@
 <!DOCTYPE html>
 <html>
-   <head>
-      <script src="jquery.js"></script>
-      <script src="rla_ui.js"></script>
-      <link rel="stylesheet" href="style.css">
-   </head>
-   <body>
-      <div id="auditTypeContainer" class="container">
-         Choose the audit type:<br />
-      </div>
-      <div id="auditNameContainer" class="container" style="display:none">
-         Enter a name for this audit:<br />
-         <input type="text" id="auditNameBox" />
-         <button id="auditNameSaveButton">Save</button>
-      </div>
-      <div id="cvrFileUploadContainer" class="container" style="display:none">
-         <p>Upload a Cast Vote Record (CVR) file</p>
-         <form id="uploadCVRForm" method="post" enctype="multipart/form-data">
+
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
```

Comment:
`audit-conductor` should be able to run without an internet connection, so we locally load e.g. jquery with:

    <script src="jquery.js"></script>

And we serve it from the app (https://github.com/ElectionAuditWare/audit-conductor/blob/3de6dca06854a035072a8fab49a984cad7d52bb0/conductor.py#L989)

If Bootstrap's important for the change, you can serve it locally the same way.